<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkDiffusionQuantificationViewControls</class>
 <widget class="QWidget" name="QmitkDiffusionQuantificationViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>286</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Q-Ball Imaging</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="m_StandardGFACheckbox">
           <property name="text">
            <string>Generalized GFA</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="m_ParamK">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>k</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_ParamKEdit">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="m_ParamP">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>p</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_ParamPEdit">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_GFAButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>GFA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Min. angle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="MinAngle"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Max. angle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="MaxAngle"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_CurvatureButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Curvature</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Tensor Imaging</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCommandLinkButton" name="m_FAButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>FA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="m_RAButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>RA</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>220</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <includes>
  <include location="local">QmitkDataStorageComboBox.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
